modules/transactions/src/main/java/org/apache/ignite/internal/tx/impl/IgniteTransactionsImpl.java

```diff
@@ -39,25 +41,31 @@ public IgniteTransactionsImpl(TxManager txManager) {
         this.txManager = txManager;
     }
 
-    /** {@inheritDoc} */
-    @Override
-    public Transaction begin(@Nullable TransactionOptions options) {
+    /**
+     * Begins a transaction.
+     *
+     * @param options Transaction options.
+     * @param observableTimestamp Timestamp.
```

Comment:
Let's minimize the usage of such placeholder descriptions :) you can easily reuse the description of observable timestamp from javadoc from here: `TxManager#begin(boolean, org.apache.ignite.internal.hlc.HybridTimestamp)`

Reply:
Good point, fixed.